app/service/document.py

```diff
@@ -92,6 +92,9 @@ def update(
     """
     validate_import_id(import_id)
 
+    if document.variant_name == "":
```

Comment:
Sorry - this is my bad, but we also need to check that `variant_name` appears in the db (see https://github.com/climatepolicyradar/navigator-admin-backend/blob/355a3dacba85b2d11c0f8571816ff803e3c98a2b/app/clients/db/models/law_policy/family.py#L23 ) ... happy this is done as a follow-up